modules/terms/partials/helm.adoc

```diff
@@ -3,4 +3,6 @@
 :hover-text: Generates and applies all the manifest files you need for deploying Redpanda in Kubernetes. 
 :category: Redpanda in Kubernetes
 
-For more information, see xref:ROOT:deploy:deployment-option/self-hosted/kubernetes/kubernetes-production-deployment.adoc[].
\ No newline at end of file
+ifndef::env-cloud[]
```

Comment:
Would we need to use this glossterm in Cloud?

Reply:
nope